packages/augur-sdk/src/state/db/DB.ts

```diff
@@ -768,4 +769,50 @@ export class DB {
     const results = await this.findInDerivedDB(this.getDatabaseName('Markets'), request);
     return results.docs as unknown as MarketData[];
   }
+
+  /**
+   * Queries the Liquidity DB for hourly liquidity of markets
+   *
+   * @param {string?} marketIds Array of market IDs to filter by
+   * @returns {Promise<MarketHourlyLiquidity[]>}
+   */
+  async findMarketsLiquidityDocs(marketIds?: string[]): Promise<MarketHourlyLiquidity[]> {
+    const currentTimestamp = new BigNumber(Math.floor(Date.now() / 1000));
```

Comment:
Should this be using block timestamp

Reply:
Hmm yeah I think I meant to change that to use the block timestamp, but I wasn't sure how to look it up here.

Reply:
I would think everything should be in ref to block time.

Reply:
Agreed, it should be. I know how to look up the highest available block number, but I'm not sure how to get the other block info.

Reply:
I think I may have figured it out. Will push a change later tonight after I've tested it out.